### PR TITLE
Support dolt.rebase for doltgres

### PR DESCRIPTION
**Pull request by tbantle22**

### CODE REVIEW

**Comment by coffeegoddd** [comment]
@tbantle22 __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>193b46f</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>193b46f</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

File: go/libraries/doltcore/sqle/database.go [hunk 1]
```diff
@@ -693,6 +693,11 @@ func (db Database) getTableInsensitive(ctx *sql.Context, head *doltdb.Commit, ds
 		return dt, found, nil
 	}
 
+	if resolve.UseSearchPath && lwrName == doltdb.RebaseTableName {
```
Comment (zachmu) [inline go/libraries/doltcore/sqle/database.go:696]: Might add a comment about what this is doing (converting `dolt_rebase` into `dolt.rebase` for compatibility

**Review by zachmu — APPROVED** [review]
LGTM

**Comment by coffeegoddd** [comment]
@tbantle22 __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>1323151</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>1323151</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Pull request merged**